src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -94,6 +95,19 @@ if (__DEV__) {
     );
   };
 
+  var warnStyleValueIsNaN = function(name, value) {
+    if (warnedStyleValues.hasOwnProperty(value) && warnedStyleValues[value]) {
```

Comment:
Please don't reuse `warnedStyleValues` - it's used for tracking semicolons in values. We can just have a global boolean `warnedForNaNValue` (we can only warn for `NaN` once).

It's unfortunate that we don't have more context in here - once and done for this warning isn't the most helpful thing (but better than nothing)